src/Integration/ETW/CodeMarkers.cs

```diff
@@ -34,6 +34,7 @@ public static class Keywords
             public const EventKeywords Binding = (EventKeywords)2;
             public const EventKeywords Analysis = (EventKeywords)4;
             public const EventKeywords CFamily = (EventKeywords)8;
+            public const EventKeywords Suppression = (EventKeywords)2;
```

Comment:
We're only recording Roslyn-related events for the moment, but we do handle suppressions for other languages. For now, I think making the key `RoslynSuppression` would be better as it will remind us in the future that it's Roslyn-specific.
The alternative is to add another keyword for `Roslyn`, but I don't think that is worth doing at the moment.